docs/contributing.rst

```diff
@@ -205,6 +205,24 @@ To run a specific test (say ``tests/test_loader.py``) use:
 
     ``tox -- tests/test_loader.py``
 
+To run the tests on a specific tox_ environment, use ``-e <name>`` with an
+environment name from ``tox.ini``. For example, to run the tests with PyPy
+use::
+
+    tox -e pypy
+
+To pass command-line options to pytest_, add them after ``--`` in your call to
+tox_. Using ``--`` overrides the default positional arguments defined in
+``tox.ini``, so you must include those default positional arguments
+(``scrapy tests``) after ``--`` as well::
+
+    tox -- scrapy tests -x  # stop after first failure
+
+If you have multiple CPU cores, you can use ``-n <number>`` to use a specific
+number of CPU cores to run the tests, or ``-n auto`` to use all of them::
+
+    tox -- scrapy tests -n auto
```

Comment:
I think in this example tests are executed for all environments; there is a different way to parallelize it (https://tox.readthedocs.io/en/latest/example/basic.html#parallel-mode). I'm not sure what is better, to run individual environments with several processes, or to run environments in parallel in this case. What do you think about having e.g. `tox -e py36 -- scrapy tests -n auto` as an example?

Reply:
You can actually use both in combination, as it turns out, but I guess it’s better to cover them separately, and let developers do what works best for them.